packages/govuk-frontend/src/govuk/components/exit-this-page/template.njk

```diff
@@ -1,5 +1,10 @@
 {% from "../button/macro.njk" import govukButton -%}
 
+{% set hasSetCustomText = params.html !== undefined or params.text !== undefined -%}
```

Comment:
There's a gotcha in this as `{ html: null }` will pass the checks

Probably too far to filter empty space with ` | trim` but will this do?

```suggestion
{% set hasSetCustomText = params.html or params.text -%}
```

Reply:
I may be misremembering (I originally wrote this about a month ago!) but I seem to recall it not working as expected without the `!== undefined`. I can give it a re-test just to be sure.

Reply:
Seems like it works fine without it, actually. I might be remembering a previous iteration, then.